spec/classes/nodejs_spec.rb

```diff
@@ -920,6 +920,16 @@
       }
     end
 
+    it 'the file resource root_npmrc should use /var/root for the path' do
+      is_expected.to contain_file('root_npmrc').with(
+        'ensure' => 'file',
+        'path'    => '/.npmrc',
```

Comment:
This doesn't match the description. Maybe this should be left out since it's already tested above.

Reply:
We could either leave it out or override the fact for this test to be closer to the default that Darwin should come up with?